### ISSUE TITLE
rlinks: create a test suite

### ISSUE THREAD

**Issue by om26er** [crossbario/crossbar#1903]
There are lots of rlinks scenario that can break when making changes to Crossbar. We need to add some testing for this stuff, so that regressions can be limited. I will come up with some ideas on how to do that.

**Comment by oberstet** [comment]
Yes, absolutely agreed! It would be good to review and list existing tests rgd rlinks. There are indeed some .. eg dynamically/programmatically setup rlinks, publish, check event is received via all nodes, and also I think master node managed rlinks ..
